rules/src/main/scala/fix/UseNamedParameters.scala

```diff
@@ -100,8 +102,10 @@ final class UseNamedParameters(config: UseNamedParametersConfig)
                 case Some(symInfo)
                     // In the case of repeated parameters, if the parameter name is given only at the beginning, it is broken.
                     if !symInfo.signature.toString().startsWith("* Tuple") =>
-                  val paramName = Term.Name(symInfo.displayName).toString
-                  Patch.addLeft(t, s"$paramName = ")
+                  Term.Name(symInfo.displayName).toString match {
+                    case pattern() if config.skipSingleAlphabet => Patch.empty
```

Comment:
I think it's more efficient if we do boolean check first before performing regex match (for those people not using skipSingleAlphabet), as the JIT/branch prediction can just optimize that codepath out.

```
val termName = ...
if (config.skipSingleAlphabet && pattern.match(....)) 
  Patch.empty
// ... etc
```

Reply:
Thank you very much! Correspondence.